### ISSUE TITLE
String interpolation `${...}` breaks CSS highlighting

### ISSUE THREAD

**Issue by zourtney** [natewallace/angular2-inline#2]
First of all, thanks for the awesome plugin -- it's a game changer!

I'm using string interpolation on `styles` to insert CSS values, as an alternative to SASS variables. But, syntax highlighting breaks after the first `${...}`. Fortunately, it seems to reset after the rule closes.

Is this fixable? I might have time to work on a pull request this weekend, but no promises.

![screen shot 2016-11-11 at 11 57 15 am](https://cloud.githubusercontent.com/assets/880060/20228523/77528538-a806-11e6-96a8-604b4f42c2ca.png)

**Setup example, for clarity**

*styles.ts*

```
export const fontSizeSmall = '12px';
```

*some.component.ts*

```
import * as styles from './styles';

@Component({
  styles: [`
    :host {
      font-size: ${styles.fontSizeSmall};
    }
  `],
  ...
```

**Comment by natewallace** [comment]
~~I'm currently using the built in css syntax highlighter.  It will be easy enough to switch over to use the built in scss syntax highlighter which should solve the problem.  I'll include this change with the next update.~~

Scratch that.  I was thinking the syntax was the same but it isn't.  I'm going to have to look at this longer as it is not an easy fix and is similar to issue #1.

**Comment by natewallace** [comment]
I've published version 0.0.15 which includes limited support for string interpolation syntax highlights.  This should fix your issue.

**Comment by zourtney** [comment]
Fix confirmed in `v0.0.16`. Thanks -- keep up the great work
